src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
+
+The System76 Eland 1U is a rack-mount server with the following specifications:
+
+- Operating System
+    - Ubuntu Server 22.04 LTS (64-bit)
+- CPU options
+    - Supports AMD EPYC™ 7003 series processor family
+    - Single processor
+- Memory
+    - Up to 4096GB (16x256GB) ECC DDR4 @ 3200 MHz
+- Networking
+    - 2x 1GbE LAN ports (1 x Intel® I350-AM2)
+    - 1x 10/100/1000 management LAN
+- Expansion
+    - 1 x PCIe x16 slot (Gen4)
+- Ports
+    - Front: 2 x USB 3.0, 1 x Power button with LED, 1 x ID button with LED, 1 x NMI button, 1 x Reset button, 2 x LAN activity LEDs, 1 x HDD activity LED, 1 x System status LED
+    - Rear: 3 x USB 3.0, 1 x VGA, 1 x COM, 2 x RJ45, 1 x MLAN, 1 x ID button with LED
+- Power Supply
+    - 2 x 650W redundant PSUs, 80 PLUS Platinum
+- Dimensions
+    - 17.2” × 26” × 1.7” (43.8cm × 66.0cm × 43.5cm)
+- Weight
+    - 44 lbs (17.24 kg)
```

Comment:
Not necessarily an issue, but we just list cm and kg for dimensions/weight on laptops (desktops are currently missing weights.) Might be good to be consistent about that, otherwise I could see someone opening an issue for all the other pages missing peoples' preferred units in the future.